_pydevd_bundle/pydevd_cython.pyx

```diff
@@ -1416,7 +1416,7 @@ def should_stop_on_exception(py_db, PyDBAdditionalThreadInfo info, frame, thread
                     pass
 
                 else:
-                    was_just_raised = trace.tb_next is None
+                    was_just_raised = just_raised(trace)
```

Comment:
Please inline this function (the difference is that it'll check if trace itself is None, just do that inline here.

If this wasn't a debugger, I'd prefer the function call, but for the debugger, additional function calls should be avoided for performance reasons -- especially here, this is all in cython and `just_raised` isn't in a cythonized module.

Reply:
Yeah this was from the change I made from django. This change isn't necessary anymore. I'll revert it.